sdk/core/core-http/lib/policies/tracingPolicy.ts

```diff
@@ -31,7 +43,20 @@ export class TracingPolicy extends BaseRequestPolicy {
 
     // create a new span
     const tracer = getTracer();
-    const span = tracer.startSpan("core-http", request.spanOptions);
+    const spanOptions: SpanOptions = {
+      ...request.spanOptions,
+      kind: SpanKind.CLIENT
+    };
+    const span = tracer.startSpan("core-http", spanOptions);
+    span.setAttributes({
+      "http.method": request.method,
+      "http.url": request.url,
```

Comment:
Perhaps this is already the case but do we have to do any redacting for things that are traced? If so - has this value already been redacted?

Reply:
We don't redact anything for tracing that I'm aware of. I can ask in the next tracing standup, but I suspect part of it is this data gets collected by the developer when they opt in and isn't automatically sent anywhere.

Reply:
right, we do not redact anything in any SDK for tracing. This was discussed on the ADP board, but there was no requirement to redact. This data is collected and sent out-of-the-box to the tracing backend/visualization tool. 

To be fair, even if you redact it, HTTP instrumenation (monkey-patching) can get hold of the unredacted string. ApplicationInsights SDK do it for years.